client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -115,6 +116,15 @@ class PaymentMethodItem extends Component {
 					onDone={ this.onDone } />
 			);
 		}
+		if ( method.id === 'cheque' ) {
```

Comment:
Looks like the others aren't yoda'd either, but maybe we should yoda all these now